muesli/web/templates/Fragments/HTML/form.pt

```diff
@@ -1,63 +1,107 @@
+          <span class="font-weight-bold text-break" id="text-input-prepend-text">${field.label}</span>
+        </div>
+        <div class="col-sm-9 col-xl-10">
+        <input type="text" class="form-control"
+        aria-describedby="text-input-prepend-text" tal:attributes="
+        name field.name; size str(field.size); value field.value;
+        aria-label field.name; readonly 'readonly' if field.readonly else None;
+        placeholder ('z.B. ' + field.placeholder) if field.placeholder else None;">
+        </div>
+      </div>
+
+      <div tal:condition="field.type=='number'" tal:omit-tag="">
+        <!-- NUMBER INPUT -->
+        <input type="number" tal:attributes="name field.name; size str(field.size); value field.value; readonly 'readonly' if field.readonly else None"/>
+      </div>
+
+      <div class="form-row" tal:condition="field.type == 'password'">
+        <!-- PASSWORD INPUT -->
+        <div class="col-sm-3 col-xl-2">
+          <span class="font-weight-bold text-break" id="password-input-prepend-text">${field.label}</span>
+        </div>
+        <div class="col-sm-9 col-xl-10">
+          <input type="password" class="form-control"
+          aria-describedby="password-input-prepend-text" tal:attributes="
+          name field.name; size str(field.size); value field.value;
+          aria-label field.name; readonly 'readonly' if field.readonly else None;">
+        </div>
+      </div>
+
+      <div class="mb-1" tal:condition="field.type=='textarea'">
+        <!-- TEXTAREA -->
+        <textarea class="form-control"
+          tal:attributes="name field.name; cols str(field.cols); rows str(field.rows); readonly 'readonly' if field.readonly else None"
+          tal:content="field.value"></textarea>
+      </div>
+
+      <div tal:condition="field.type=='file'" tal:omit-tag="">
+        <!-- FILE SELECTION -->
+        <div class="custom-file w-25">
+          <input type="file" class="custom-file-input" id="file_upload"
+          tal:attributes="name field.name; size str(field.size); value field.value" />
+          <label class="custom-file-label" for="file_upload">Choose file</label>
+        </div>
+      </div>
+
+      <div tal:condition="field.type=='multifile'" tal:omit-tag="">
+        <!-- MULTIFILE SELECTION -->
+        <input type="file" tal:attributes="name field.name; size str(field.size); value field.value" multiple/>
+      </div>
+
+
+      <div name="formerror" class="alert alert-danger" tal:condition="field.name in form.errors"
+          tal:content="'&uarr; ' + form.errors[field.name]">
+      </div>
+    </div>
+
+    <div tal:condition="field.type == 'hidden'" tal:omit-tag="">
+      <!-- HIDDEN INPUT -->
+      <input type="hidden" tal:attributes="name field.name; size str(field.size); value field.value; readonly 'readonly' if field.readonly else None"/>
+    </div>
+
+  </div>
+  <input role="button" class="btn btn-primary" type="submit" tal:attributes="value form.send" />
+</div>
```

Comment:
> I like your comments. They are so helpful :D

\*coughs\* not that I added them because I got lost in the resulted HTML code for too many times xD